2018-06-26 14:54:17,061 ERROR: Exception on /edit_profile [POST] [in /Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-06-26 14:54:48,915 ERROR: Exception on /edit_profile [POST] [in /Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-06-26 14:55:11,873 ERROR: Exception on /edit_profile [POST] [in /Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/ripzy/Projects/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-06-26 15:03:04,528 INFO: Microblog startup [in /Users/ripzy/Projects/microblog/app/__init__.py:47]
2018-06-26 15:37:10,228 INFO: Microblog startup [in /Users/ripzy/Projects/microblog/app/__init__.py:47]
